target/linux/bcm27xx/patches-5.15/950-0600-xhci-quirks-add-link-TRB-quirk-for-VL805.patch

```diff
@@ -48,13 +48,3 @@ Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
  
  	/* Don't update the ring cycle state for the producer (us). */
  	addr = xhci_trb_virt_to_dma(new_seg, new_deq);
---- a/drivers/usb/host/xhci.h
-+++ b/drivers/usb/host/xhci.h
-@@ -1903,6 +1903,7 @@ struct xhci_hcd {
- #define XHCI_NO_SOFT_RETRY	BIT_ULL(40)
- #define XHCI_BROKEN_D3COLD	BIT_ULL(41)
- #define XHCI_EP_CTX_BROKEN_DCS	BIT_ULL(42)
-+#define XHCI_AVOID_DQ_ON_LINK	BIT_ULL(43)
```

Comment:
This is wrong `XHCI_AVOID_DQ_ON_LINK` was not added in v5.15.75, but XHCI_SUSPEND_RESUME_CLKS was added. See https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?id=f002aa7c0ac5518a152ec24503c9f887031984a0 This change is still needed, but you have to it a new number , increase it by 1. There are 2 other files with this problem.

Reply:
Yes!  This is what I did not know.  So keep the numbers from upstream, then add the next largest number for this. I will do that and rebase.

Reply:
@hauke @Ansuel - Please review my latest push wherein I believe I corrected this mistake.

Reply:
I am going to mark this as resolved.  Boot on my RPi4 with no dmesg regressions/USB seems to operate as expected.

Reply:
looks good now